ttnn/cpp/ttnn/operations/eltwise/binary/device/broadcast_width_multi_core_program_factory.cpp

```diff
@@ -87,15 +87,15 @@ BinaryDeviceOperation::BroadcastWidthMultiCore::cached_program_t BinaryDeviceOpe
     auto dst_buffer = output.buffer();
     TT_ASSERT(dst_buffer != nullptr, "Output buffer should be allocated on device!");
 
-    uint32_t src0_cb_index = 0;
+    uint32_t src0_cb_index = GetNextAvailableCircularBufferIndex(program);
     uint32_t num_input_tiles = 2;
 
     tt_metal::CircularBufferConfig src0_cb_config =
         tt_metal::CircularBufferConfig(num_input_tiles * src0_single_tile_size, {{src0_cb_index, src0_cb_data_format}})
             .set_page_size(src0_cb_index, src0_single_tile_size);
     auto cb_src0 = tt_metal::CreateCircularBuffer(program, all_device_cores, src0_cb_config);
 
-    uint32_t src1_cb_index = 1;
+    uint32_t src1_cb_index = GetNextAvailableCircularBufferIndex(program);;
```

Comment:
From examples I see in the PR is it looks like Op developers don't use this index. 

I wonder if it possible to assign this index in CreateCircularBuffer w/o even exposing it to a user?

Reply:
It's used when defining the CircularBufferConfig.  The idea is to be able to have different possible setups for the same CB if you wanted to.  In this case they don't need to.
```
        uint32_t src1_cb_index = GetNextAvailableCircularBufferIndex(program);
    tt_metal::CircularBufferConfig src1_cb_config =
        tt_metal::CircularBufferConfig(num_input_tiles * src1_single_tile_size, {{src1_cb_index, src1_cb_data_format}})
            .set_page_size(src1_cb_index, src1_single_tile_size);
    auto cb_src1 = tt_metal::CreateCircularBuffer(program, all_device_cores, src1_cb_config);
```